<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="Dock">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>96</width>
      <height>480</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>96</width>
      <height>480</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>96</width>
       <height>81</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>96</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>96</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>94</width>
        <height>79</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="SideBar">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>200</width>
      <height>480</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>480</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>16777215</height>
     </size>
    </property>
    <widget class="QWidget" name="page"/>
    <widget class="QWidget" name="page_2"/>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>0</y>
      <width>16</width>
      <height>491</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Content">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>661</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="ToolBar">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>661</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QToolButton" name="UndoBtn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::EditUndo"/>
     </property>
    </widget>
    <widget class="QToolButton" name="RedoBtn">
     <property name="geometry">
      <rect>
       <x>32</x>
       <y>0</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::EditRedo"/>
     </property>
    </widget>
    <widget class="QToolButton" name="CopyBtn">
     <property name="geometry">
      <rect>
       <x>64</x>
       <y>0</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::EditCopy"/>
     </property>
    </widget>
    <widget class="QToolButton" name="CutBtn">
     <property name="geometry">
      <rect>
       <x>96</x>
       <y>0</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::EditCut"/>
     </property>
    </widget>
    <widget class="QToolButton" name="PasteBtn">
     <property name="geometry">
      <rect>
       <x>128</x>
       <y>0</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::EditPaste"/>
     </property>
    </widget>
    <widget class="QToolButton" name="FindBtn">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>0</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::EditFind"/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>224</x>
       <y>0</y>
       <width>256</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action514">
   <property name="text">
    <string>514</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
